src/index.js

```diff
@@ -1,10 +1,35 @@
 'use strict';
 /**
  * @module braintree-web-drop-in
+ * @description There are two ways to integrate Drop-in into your page. You can use [`dropin.create`](#.create) or a standalone script tag integration (example below).
```

Comment:
I think we could mention it briefly on the homepage as well. I could see us emphasizing `dropin.create` as the main way to integrate with the script tag as an alternative, where this kinda reads like the script tag integration is the default.